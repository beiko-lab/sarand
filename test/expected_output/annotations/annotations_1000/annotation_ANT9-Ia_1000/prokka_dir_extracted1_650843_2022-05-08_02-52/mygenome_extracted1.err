Discrepancy Report Results

Summary
FATAL: MISSING_PROTEIN_ID:4 proteins have invalid IDs.
DISC_SOURCE_QUALS_ASNDISC:strain (all present, all unique)
DISC_SOURCE_QUALS_ASNDISC:taxname (all present, all unique)
DISC_FEATURE_COUNT:CDS: 4 present
DISC_COUNT_NUCLEOTIDES:1 nucleotide Bioseqs are present
FEATURE_LOCATION_CONFLICT:4 features have inconsistent gene locations.
DISC_QUALITY_SCORES:Quality scores are missing on all sequences.
ONCALLER_COMMENT_PRESENT:1 comment descriptors were found (all same)
MISSING_GENOMEASSEMBLY_COMMENTS:1 bioseqs are missing GenomeAssembly structured comments
MOLTYPE_NOT_MRNA:1 molecule types are not set as mRNA.
TECHNIQUE_NOT_TSA:1 technique are not set as TSA
MISSING_STRUCTURED_COMMENT:1 sequences do not include structured comments.
MISSING_PROJECT:5 sequences do not include project.
DISC_INCONSISTENT_MOLINFO_TECH:Molinfo Technique Report (some missing, all same)


Detailed Report

FATAL: DiscRep_ALL:MISSING_PROTEIN_ID::4 proteins have invalid IDs.
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence:_1 (length 151)
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence:_2 (length 125)
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence:_3 (length 260)
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence:_4 (length 243)

DiscRep_ALL:DISC_SOURCE_QUALS_ASNDISC::strain (all present, all unique)
DiscRep_SUB:DISC_SOURCE_QUALS_ASNDISC::1 sources have unique values for strain
DiscRep_ALL:DISC_SOURCE_QUALS_ASNDISC::taxname (all present, all unique)
DiscRep_SUB:DISC_SOURCE_QUALS_ASNDISC::1 sources have unique values for taxname
DiscRep_ALL:DISC_FEATURE_COUNT::CDS: 4 present
DiscRep_ALL:DISC_COUNT_NUCLEOTIDES::1 nucleotide Bioseqs are present
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2783)

DiscRep_ALL:FEATURE_LOCATION_CONFLICT::4 features have inconsistent gene locations.
DiscRep_SUB:FEATURE_LOCATION_CONFLICT::Coding region xref gene does not exist
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:CDS	hypothetical protein	sequence::11-466	FBNJAKFI_00001

DiscRep_SUB:FEATURE_LOCATION_CONFLICT::Coding region xref gene does not exist
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:CDS	hypothetical protein	sequence::473-850	FBNJAKFI_00002

DiscRep_SUB:FEATURE_LOCATION_CONFLICT::Coding region xref gene does not exist
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:CDS	Streptomycin 3''-adenylyltransferase	sequence::1001-1783	FBNJAKFI_00003

DiscRep_SUB:FEATURE_LOCATION_CONFLICT::Coding region xref gene does not exist
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:CDS	rRNA adenine N-6-methyltransferase	sequence::c2640-1909	FBNJAKFI_00004

DiscRep_ALL:DISC_QUALITY_SCORES::Quality scores are missing on all sequences.

DiscRep_ALL:ONCALLER_COMMENT_PRESENT::1 comment descriptors were found (all same)
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence::Annotated using prokka 1.14.5 from https://github.com/tseemann/prokka

DiscRep_ALL:MISSING_GENOMEASSEMBLY_COMMENTS::1 bioseqs are missing GenomeAssembly structured comments
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2783)

DiscRep_ALL:MOLTYPE_NOT_MRNA::1 molecule types are not set as mRNA.
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2783)

DiscRep_ALL:TECHNIQUE_NOT_TSA::1 technique are not set as TSA
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2783)

DiscRep_ALL:MISSING_STRUCTURED_COMMENT::1 sequences do not include structured comments.
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2783)

DiscRep_ALL:MISSING_PROJECT::5 sequences do not include project.
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2783)
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence:_1 (length 151)
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence:_2 (length 125)
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence:_3 (length 260)
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence:_4 (length 243)

DiscRep_ALL:DISC_INCONSISTENT_MOLINFO_TECH::Molinfo Technique Report (some missing, all same)
DiscRep_SUB:DISC_INCONSISTENT_MOLINFO_TECH::technique (all missing)
DiscRep_SUB:DISC_INCONSISTENT_MOLINFO_TECH::1 Molinfos are missing field technique
prokka_dir_extracted1_650843_2022-05-08_02-52/mygenome_extracted1:sequence: (length 2783)

