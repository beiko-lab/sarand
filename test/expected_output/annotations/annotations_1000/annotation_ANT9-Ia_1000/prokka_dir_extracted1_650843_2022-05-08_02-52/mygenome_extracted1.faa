>FBNJAKFI_00001 hypothetical protein
MPQHIVQKFLGHESPEMTSRYAHIFDETLKNEFTKFQEKLVTNNGDVLDLDEDNEVDDVE
LQWFKKNINAQVLPNGYCRLPVVAGGCPHANACLDCTHFCTSKQFLPQHEEQLERTEELL
AIAKDKQWQRQVETNSRVKERLEQIIGSLTG
>FBNJAKFI_00002 hypothetical protein
MDKQVRNTTEIVRLAKQKSKKTREKVDKAISKFSIEGKVINFNSIAKEANVSKSWLYKEH
DIRQRIESLRERQITANVVSKPKKSSRSEEILIKTLKRRVMELEKENKKLQNQIQKLYGD
LYNKE
>FBNJAKFI_00003 Streptomycin 3''-adenylyltransferase
MSNLINGKIPNQAIQTLKIVKDLFGSSIVGVYLFGSAVNGGLRINSDVDVLVVVNHSLPQ
LTRKKLTERLMTISGKIGNTDSVRPLEVTVINRSEVVPWQYPPKREFIYGEWLRGEFENG
QIQEPSYDPDLAIVLAQARKNSISLFGPDSSSILVSVPLTDIRRAIKDSLPELIEGIKGD
ERNVILTLARMWQTVTTGEITSKDVAAEWAIPLLPKEHVTLLDIARKGYRGECDDKWEGL
YSKVKALVKYMKNSIETSLN
>FBNJAKFI_00004 rRNA adenine N-6-methyltransferase
MNQKNPKDTQNFITSKKHVKEILNHTNISKQDNVIEIGSGKGHFTKELVKMSRSVTAIEI
DGGLCQVTKEAVNPSENIKVIQTDILKFSFPKHINYKIYGNIPYNISTDIVKRITFESQA
KYSYLIVEKGFAKRLQNLQRALGLLLMVEMDIKMLKKVPPLYFHPKPSVDSVLIVLERHQ
PLISKKDYKKYRSFVYKWVNREYRVLFTKNQFRQALKHANVTNINKLSKEQFLSIFNSYK
LFH
