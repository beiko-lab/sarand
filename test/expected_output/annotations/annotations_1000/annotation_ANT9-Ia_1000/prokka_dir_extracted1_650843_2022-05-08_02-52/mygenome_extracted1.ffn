>FBNJAKFI_00001 hypothetical protein
ATGCCCCAGCATATTGTGCAGAAATTTTTGGGGCATGAAAGCCCAGAAATGACAAGCAGA
TACGCTCATATCTTTGATGAAACTCTAAAAAATGAATTTACTAAATTTCAGGAAAAACTG
GTTACCAATAATGGAGATGTGCTTGATCTAGATGAAGATAATGAAGTCGATGATGTAGAG
CTTCAATGGTTCAAGAAAAATATAAATGCACAAGTGCTTCCAAATGGTTATTGTAGATTG
CCAGTAGTAGCAGGTGGTTGTCCACATGCGAATGCATGCTTAGATTGCACTCACTTCTGT
ACCAGTAAGCAATTCTTACCACAGCACGAAGAACAGTTAGAGCGTACAGAAGAGTTATTA
GCCATAGCTAAGGATAAACAATGGCAAAGACAAGTAGAGACTAATAGCCGTGTTAAAGAG
CGTTTAGAACAAATCATTGGAAGTTTGACGGGGTAA
>FBNJAKFI_00002 hypothetical protein
ATGGATAAACAAGTTAGAAATACAACAGAAATTGTACGTTTGGCGAAGCAGAAATCAAAA
AAGACAAGGGAAAAAGTAGACAAAGCGATTTCTAAATTTTCGATTGAAGGTAAAGTTATT
AATTTTAATTCAATAGCAAAGGAAGCTAATGTTTCTAAATCATGGCTTTATAAGGAACAC
GATATTAGGCAAAGAATCGAATCCCTTCGTGAGCGTCAAATAACAGCAAATGTAGTCTCA
AAACCCAAGAAAAGTTCTCGTTCGGAGGAAATCCTTATTAAAACCTTAAAAAGAAGAGTA
ATGGAATTAGAAAAAGAAAATAAAAAATTACAGAACCAAATTCAAAAATTATATGGAGAT
CTGTATAATAAAGAATAA
>FBNJAKFI_00003 Streptomycin 3''-adenylyltransferase
ATGAGCAATTTGATTAACGGAAAAATACCAAATCAAGCGATTCAAACATTAAAAATCGTA
AAAGATTTATTTGGAAGTTCAATAGTTGGAGTATATCTATTTGGTTCAGCAGTAAATGGT
GGTTTACGCATTAACAGCGATGTAGATGTTCTAGTCGTCGTGAATCATAGTTTACCTCAA
TTAACTCGAAAAAAACTAACAGAAAGACTAATGACTATATCAGGAAAGATTGGAAATACG
GATTCTGTTAGACCACTTGAAGTTACGGTTATAAATAGGAGTGAAGTTGTCCCTTGGCAA
TATCCTCCAAAAAGAGAATTTATATACGGTGAGTGGCTCAGGGGTGAATTTGAGAATGGA
CAAATTCAGGAACCAAGCTATGATCCTGATTTGGCTATTGTTTTAGCACAAGCAAGAAAG
AATAGTATTTCTCTATTTGGTCCTGATTCTTCAAGTATACTTGTCTCCGTACCTTTGACA
GATATTCGAAGAGCAATTAAGGATTCTTTGCCAGAACTAATTGAGGGGATAAAAGGTGAT
GAGCGTAATGTAATTTTAACCCTAGCTCGAATGTGGCAAACAGTGACTACTGGTGAAATT
ACCTCGAAAGATGTCGCTGCAGAATGGGCTATACCTCTTTTACCTAAAGAGCATGTAACT
TTACTGGATATAGCTAGAAAAGGCTATCGGGGAGAGTGTGATGATAAGTGGGAAGGACTA
TATTCAAAGGTGAAAGCACTCGTTAAGTATATGAAAAATTCTATAGAAACTTCTCTCAAT
TAG
>FBNJAKFI_00004 rRNA adenine N-6-methyltransferase
ATGAACCAGAAAAACCCTAAAGACACGCAAAATTTTATTACTTCTAAAAAGCATGTAAAA
GAAATATTGAATCACACGAATATCAGTAAACAAGACAACGTAATAGAAATCGGATCAGGA
AAAGGACATTTTACCAAAGAGCTAGTCAAAATGAGTCGATCAGTTACTGCTATAGAAATT
GATGGAGGCTTATGTCAAGTGACTAAAGAAGCGGTAAACCCCTCTGAGAATATAAAAGTG
ATTCAAACGGATATTCTAAAATTTTCCTTCCCAAAACATATAAACTATAAGATATATGGT
AATATTCCTTATAACATCAGTACGGATATTGTCAAAAGAATTACCTTTGAAAGTCAGGCT
AAATATAGCTATCTTATCGTTGAGAAGGGATTTGCGAAAAGATTGCAAAATCTGCAACGA
GCTTTGGGTTTACTATTAATGGTGGAGATGGATATAAAAATGCTCAAAAAAGTACCACCA
CTATATTTTCATCCTAAGCCAAGTGTAGACTCTGTATTGATTGTTCTTGAACGACATCAA
CCATTGATTTCAAAGAAGGACTACAAAAAGTATCGATCTTTTGTTTATAAGTGGGTAAAC
CGTGAATATCGTGTTCTTTTCACTAAAAACCAATTCCGACAGGCTTTGAAGCATGCAAAT
GTCACTAATATTAATAAACTATCGAAGGAACAATTTCTTTCTATTTTCAATAGTTACAAA
TTGTTTCACTAA
