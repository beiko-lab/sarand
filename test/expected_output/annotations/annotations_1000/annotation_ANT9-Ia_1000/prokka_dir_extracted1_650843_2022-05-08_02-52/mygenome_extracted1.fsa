>sequence: [gcode=11] [organism=Genus species] [strain=strain]
TAACAACGGGATGCCCCAGCATATTGTGCAGAAATTTTTGGGGCATGAAAGCCCAGAAAT
GACAAGCAGATACGCTCATATCTTTGATGAAACTCTAAAAAATGAATTTACTAAATTTCA
GGAAAAACTGGTTACCAATAATGGAGATGTGCTTGATCTAGATGAAGATAATGAAGTCGA
TGATGTAGAGCTTCAATGGTTCAAGAAAAATATAAATGCACAAGTGCTTCCAAATGGTTA
TTGTAGATTGCCAGTAGTAGCAGGTGGTTGTCCACATGCGAATGCATGCTTAGATTGCAC
TCACTTCTGTACCAGTAAGCAATTCTTACCACAGCACGAAGAACAGTTAGAGCGTACAGA
AGAGTTATTAGCCATAGCTAAGGATAAACAATGGCAAAGACAAGTAGAGACTAATAGCCG
TGTTAAAGAGCGTTTAGAACAAATCATTGGAAGTTTGACGGGGTAATTATCAATGGATAA
ACAAGTTAGAAATACAACAGAAATTGTACGTTTGGCGAAGCAGAAATCAAAAAAGACAAG
GGAAAAAGTAGACAAAGCGATTTCTAAATTTTCGATTGAAGGTAAAGTTATTAATTTTAA
TTCAATAGCAAAGGAAGCTAATGTTTCTAAATCATGGCTTTATAAGGAACACGATATTAG
GCAAAGAATCGAATCCCTTCGTGAGCGTCAAATAACAGCAAATGTAGTCTCAAAACCCAA
GAAAAGTTCTCGTTCGGAGGAAATCCTTATTAAAACCTTAAAAAGAAGAGTAATGGAATT
AGAAAAAGAAAATAAAAAATTACAGAACCAAATTCAAAAATTATATGGAGATCTGTATAA
TAAAGAATAATTATTAATCTGTAGACAAATTGTGAAAGGATGTACTTAAACGCTAACGGT
CAGCTTTATTGAACAGTAATTTAAGTATATGTCCAATCTAGGGTAAGTAAATTGAGTATC
AATATAAACTTTATATGAACATAATCAACGAGGTGAAATCATGAGCAATTTGATTAACGG
AAAAATACCAAATCAAGCGATTCAAACATTAAAAATCGTAAAAGATTTATTTGGAAGTTC
AATAGTTGGAGTATATCTATTTGGTTCAGCAGTAAATGGTGGTTTACGCATTAACAGCGA
TGTAGATGTTCTAGTCGTCGTGAATCATAGTTTACCTCAATTAACTCGAAAAAAACTAAC
AGAAAGACTAATGACTATATCAGGAAAGATTGGAAATACGGATTCTGTTAGACCACTTGA
AGTTACGGTTATAAATAGGAGTGAAGTTGTCCCTTGGCAATATCCTCCAAAAAGAGAATT
TATATACGGTGAGTGGCTCAGGGGTGAATTTGAGAATGGACAAATTCAGGAACCAAGCTA
TGATCCTGATTTGGCTATTGTTTTAGCACAAGCAAGAAAGAATAGTATTTCTCTATTTGG
TCCTGATTCTTCAAGTATACTTGTCTCCGTACCTTTGACAGATATTCGAAGAGCAATTAA
GGATTCTTTGCCAGAACTAATTGAGGGGATAAAAGGTGATGAGCGTAATGTAATTTTAAC
CCTAGCTCGAATGTGGCAAACAGTGACTACTGGTGAAATTACCTCGAAAGATGTCGCTGC
AGAATGGGCTATACCTCTTTTACCTAAAGAGCATGTAACTTTACTGGATATAGCTAGAAA
AGGCTATCGGGGAGAGTGTGATGATAAGTGGGAAGGACTATATTCAAAGGTGAAAGCACT
CGTTAAGTATATGAAAAATTCTATAGAAACTTCTCTCAATTAGGCTAATTTTATTGCAAT
AACAGGTGCTTACTTTTAAAACTACTGATTTATTGATAAATATTGAACAATTTTTGGGAA
GAATAAAGCGTCCTCTTGTGAAATTAGAGAACGCTTTATTACTTTAATTTAGTGAAACAA
TTTGTAACTATTGAAAATAGAAAGAAATTGTTCCTTCGATAGTTTATTAATATTAGTGAC
ATTTGCATGCTTCAAAGCCTGTCGGAATTGGTTTTTAGTGAAAAGAACACGATATTCACG
GTTTACCCACTTATAAACAAAAGATCGATACTTTTTGTAGTCCTTCTTTGAAATCAATGG
TTGATGTCGTTCAAGAACAATCAATACAGAGTCTACACTTGGCTTAGGATGAAAATATAG
TGGTGGTACTTTTTTGAGCATTTTTATATCCATCTCCACCATTAATAGTAAACCCAAAGC
TCGTTGCAGATTTTGCAATCTTTTCGCAAATCCCTTCTCAACGATAAGATAGCTATATTT
AGCCTGACTTTCAAAGGTAATTCTTTTGACAATATCCGTACTGATGTTATAAGGAATATT
ACCATATATCTTATAGTTTATATGTTTTGGGAAGGAAAATTTTAGAATATCCGTTTGAAT
CACTTTTATATTCTCAGAGGGGTTTACCGCTTCTTTAGTCACTTGACATAAGCCTCCATC
AATTTCTATAGCAGTAACTGATCGACTCATTTTGACTAGCTCTTTGGTAAAATGTCCTTT
TCCTGATCCGATTTCTATTACGTTGTCTTGTTTACTGATATTCGTGTGATTCAATATTTC
TTTTACATGCTTTTTAGAAGTAATAAAATTTTGCGTGTCTTTAGGGTTTTTCTGGTTCAT
TATAACCTTCTCCTTACTGGTTCATTATAACCTTCTCCTTACTGGTTATAATGAACTGAC
ATTAAGCGTTCATTATAACCAATTAATTTTGATTTGGTTGATAACGAACTTTATTAATAA
CAAATATAGAAAAAGTACCCATA
