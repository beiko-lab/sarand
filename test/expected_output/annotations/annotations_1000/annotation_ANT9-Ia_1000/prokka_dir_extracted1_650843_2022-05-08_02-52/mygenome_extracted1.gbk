LOCUS       sequence:               2783 bp    DNA     linear       08-MAY-2022
DEFINITION  Genus species strain strain.
ACCESSION   
VERSION
KEYWORDS    .
SOURCE      Genus species
  ORGANISM  Genus species
            Unclassified.
COMMENT     Annotated using prokka 1.14.5 from
            https://github.com/tseemann/prokka.
FEATURES             Location/Qualifiers
     source          1..2783
                     /organism="Genus species"
                     /mol_type="genomic DNA"
                     /strain="strain"
     CDS             11..466
                     /locus_tag="FBNJAKFI_00001"
                     /inference="ab initio prediction:Prodigal:002006"
                     /codon_start=1
                     /transl_table=11
                     /product="hypothetical protein"
                     /translation="MPQHIVQKFLGHESPEMTSRYAHIFDETLKNEFTKFQEKLVTNN
                     GDVLDLDEDNEVDDVELQWFKKNINAQVLPNGYCRLPVVAGGCPHANACLDCTHFCTS
                     KQFLPQHEEQLERTEELLAIAKDKQWQRQVETNSRVKERLEQIIGSLTG"
     CDS             473..850
                     /locus_tag="FBNJAKFI_00002"
                     /inference="ab initio prediction:Prodigal:002006"
                     /codon_start=1
                     /transl_table=11
                     /product="hypothetical protein"
                     /translation="MDKQVRNTTEIVRLAKQKSKKTREKVDKAISKFSIEGKVINFNS
                     IAKEANVSKSWLYKEHDIRQRIESLRERQITANVVSKPKKSSRSEEILIKTLKRRVME
                     LEKENKKLQNQIQKLYGDLYNKE"
     CDS             1001..1783
                     /gene="ant1"
                     /locus_tag="FBNJAKFI_00003"
                     /EC_number="2.7.7.47"
                     /inference="ab initio prediction:Prodigal:002006"
                     /inference="similar to AA sequence:UniProtKB:P0A0D1"
                     /codon_start=1
                     /transl_table=11
                     /product="Streptomycin 3''-adenylyltransferase"
                     /translation="MSNLINGKIPNQAIQTLKIVKDLFGSSIVGVYLFGSAVNGGLRI
                     NSDVDVLVVVNHSLPQLTRKKLTERLMTISGKIGNTDSVRPLEVTVINRSEVVPWQYP
                     PKREFIYGEWLRGEFENGQIQEPSYDPDLAIVLAQARKNSISLFGPDSSSILVSVPLT
                     DIRRAIKDSLPELIEGIKGDERNVILTLARMWQTVTTGEITSKDVAAEWAIPLLPKEH
                     VTLLDIARKGYRGECDDKWEGLYSKVKALVKYMKNSIETSLN"
     CDS             complement(1909..2640)
                     /gene="ermA1"
                     /locus_tag="FBNJAKFI_00004"
                     /EC_number="2.1.1.184"
                     /inference="ab initio prediction:Prodigal:002006"
                     /inference="similar to AA sequence:UniProtKB:P0A0H2"
                     /codon_start=1
                     /transl_table=11
                     /product="rRNA adenine N-6-methyltransferase"
                     /translation="MNQKNPKDTQNFITSKKHVKEILNHTNISKQDNVIEIGSGKGHF
                     TKELVKMSRSVTAIEIDGGLCQVTKEAVNPSENIKVIQTDILKFSFPKHINYKIYGNI
                     PYNISTDIVKRITFESQAKYSYLIVEKGFAKRLQNLQRALGLLLMVEMDIKMLKKVPP
                     LYFHPKPSVDSVLIVLERHQPLISKKDYKKYRSFVYKWVNREYRVLFTKNQFRQALKH
                     ANVTNINKLSKEQFLSIFNSYKLFH"
ORIGIN      
        1 taacaacggg atgccccagc atattgtgca gaaatttttg gggcatgaaa gcccagaaat
       61 gacaagcaga tacgctcata tctttgatga aactctaaaa aatgaattta ctaaatttca
      121 ggaaaaactg gttaccaata atggagatgt gcttgatcta gatgaagata atgaagtcga
      181 tgatgtagag cttcaatggt tcaagaaaaa tataaatgca caagtgcttc caaatggtta
      241 ttgtagattg ccagtagtag caggtggttg tccacatgcg aatgcatgct tagattgcac
      301 tcacttctgt accagtaagc aattcttacc acagcacgaa gaacagttag agcgtacaga
      361 agagttatta gccatagcta aggataaaca atggcaaaga caagtagaga ctaatagccg
      421 tgttaaagag cgtttagaac aaatcattgg aagtttgacg gggtaattat caatggataa
      481 acaagttaga aatacaacag aaattgtacg tttggcgaag cagaaatcaa aaaagacaag
      541 ggaaaaagta gacaaagcga tttctaaatt ttcgattgaa ggtaaagtta ttaattttaa
      601 ttcaatagca aaggaagcta atgtttctaa atcatggctt tataaggaac acgatattag
      661 gcaaagaatc gaatcccttc gtgagcgtca aataacagca aatgtagtct caaaacccaa
      721 gaaaagttct cgttcggagg aaatccttat taaaacctta aaaagaagag taatggaatt
      781 agaaaaagaa aataaaaaat tacagaacca aattcaaaaa ttatatggag atctgtataa
      841 taaagaataa ttattaatct gtagacaaat tgtgaaagga tgtacttaaa cgctaacggt
      901 cagctttatt gaacagtaat ttaagtatat gtccaatcta gggtaagtaa attgagtatc
      961 aatataaact ttatatgaac ataatcaacg aggtgaaatc atgagcaatt tgattaacgg
     1021 aaaaatacca aatcaagcga ttcaaacatt aaaaatcgta aaagatttat ttggaagttc
     1081 aatagttgga gtatatctat ttggttcagc agtaaatggt ggtttacgca ttaacagcga
     1141 tgtagatgtt ctagtcgtcg tgaatcatag tttacctcaa ttaactcgaa aaaaactaac
     1201 agaaagacta atgactatat caggaaagat tggaaatacg gattctgtta gaccacttga
     1261 agttacggtt ataaatagga gtgaagttgt cccttggcaa tatcctccaa aaagagaatt
     1321 tatatacggt gagtggctca ggggtgaatt tgagaatgga caaattcagg aaccaagcta
     1381 tgatcctgat ttggctattg ttttagcaca agcaagaaag aatagtattt ctctatttgg
     1441 tcctgattct tcaagtatac ttgtctccgt acctttgaca gatattcgaa gagcaattaa
     1501 ggattctttg ccagaactaa ttgaggggat aaaaggtgat gagcgtaatg taattttaac
     1561 cctagctcga atgtggcaaa cagtgactac tggtgaaatt acctcgaaag atgtcgctgc
     1621 agaatgggct atacctcttt tacctaaaga gcatgtaact ttactggata tagctagaaa
     1681 aggctatcgg ggagagtgtg atgataagtg ggaaggacta tattcaaagg tgaaagcact
     1741 cgttaagtat atgaaaaatt ctatagaaac ttctctcaat taggctaatt ttattgcaat
     1801 aacaggtgct tacttttaaa actactgatt tattgataaa tattgaacaa tttttgggaa
     1861 gaataaagcg tcctcttgtg aaattagaga acgctttatt actttaattt agtgaaacaa
     1921 tttgtaacta ttgaaaatag aaagaaattg ttccttcgat agtttattaa tattagtgac
     1981 atttgcatgc ttcaaagcct gtcggaattg gtttttagtg aaaagaacac gatattcacg
     2041 gtttacccac ttataaacaa aagatcgata ctttttgtag tccttctttg aaatcaatgg
     2101 ttgatgtcgt tcaagaacaa tcaatacaga gtctacactt ggcttaggat gaaaatatag
     2161 tggtggtact tttttgagca tttttatatc catctccacc attaatagta aacccaaagc
     2221 tcgttgcaga ttttgcaatc ttttcgcaaa tcccttctca acgataagat agctatattt
     2281 agcctgactt tcaaaggtaa ttcttttgac aatatccgta ctgatgttat aaggaatatt
     2341 accatatatc ttatagttta tatgttttgg gaaggaaaat tttagaatat ccgtttgaat
     2401 cacttttata ttctcagagg ggtttaccgc ttctttagtc acttgacata agcctccatc
     2461 aatttctata gcagtaactg atcgactcat tttgactagc tctttggtaa aatgtccttt
     2521 tcctgatccg atttctatta cgttgtcttg tttactgata ttcgtgtgat tcaatatttc
     2581 ttttacatgc tttttagaag taataaaatt ttgcgtgtct ttagggtttt tctggttcat
     2641 tataaccttc tccttactgg ttcattataa ccttctcctt actggttata atgaactgac
     2701 attaagcgtt cattataacc aattaatttt gatttggttg ataacgaact ttattaataa
     2761 caaatataga aaaagtaccc ata
//
