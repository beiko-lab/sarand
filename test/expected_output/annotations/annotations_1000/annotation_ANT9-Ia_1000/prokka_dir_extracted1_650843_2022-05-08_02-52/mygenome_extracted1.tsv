locus_tag	ftype	length_bp	gene	EC_number	COG	product
FBNJAKFI_00001	CDS	456				hypothetical protein
FBNJAKFI_00002	CDS	378				hypothetical protein
FBNJAKFI_00003	CDS	783	ant1	2.7.7.47		Streptomycin 3''-adenylyltransferase
FBNJAKFI_00004	CDS	732	ermA1	2.1.1.184		rRNA adenine N-6-methyltransferase
